packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -323,16 +323,16 @@ async function generateBase64({ file, args = {}, reporter }) {
     .png({
       compressionLevel: options.pngCompressionLevel,
       adaptiveFiltering: false,
-      force: args.toFormat === `png`,
+      force: options.toFormat === `png`,
```

Comment:
better compare case insensitive?

Reply:
It's handled earlier in the `healOptions()` call:

https://github.com/gatsbyjs/gatsby/blob/1419afe21a48fc4d75909e054760693fadc8675f/packages/gatsby-plugin-sharp/src/plugin-options.js#L83-L84